vars/odsComponentStageScanWithSnyk.groovy

```diff
@@ -0,0 +1,15 @@
+import org.ods.component.ScanWithSnykStage
+import org.ods.component.IContext
+
+import org.ods.services.SnykService
+import org.ods.services.ServiceRegistry
+
+def call(IContext context, Map config = [:]) {
+    def snykService = ServiceRegistry.instance.get(SnykService)
```

Comment:
Not sure if I missed it but I couldn't find where you add this to the service registry?

Reply:
You are right, I'm not adding it before. Except in the tests ... test induced damage I guess :( Maybe better to test just the class - I'll think about that ...